src/library_gl.js

```diff
@@ -715,9 +715,22 @@ var LibraryGL = {
     switch(name_) {
       case 0x1F00 /* GL_VENDOR */:
       case 0x1F01 /* GL_RENDERER */:
-      case 0x1F02 /* GL_VERSION */:
         ret = allocate(intArrayFromString(GLctx.getParameter(name_)), 'i8', ALLOC_NORMAL);
         break;
+      case 0x1F02 /* GL_VERSION */:
+        var glVersion = GLctx.getParameter(GLctx.VERSION);
+        // return GLES version string corresponding to the version of the WebGL context
+#if USE_WEBGL2
+        if (GLctx.canvas.GLctxObject.version >= 2) {
```

Comment:
Oh, now I see after reading your PR comment, that not trusting the information given by the browser was exactly your intent here because of Edge's 0.96. (they also had WebGL 0.92 in some version of Edge in the past, not sure if other numbers as well).

I'm not sure which way to go here. The gut feeling is that one browser reporting out of line version result should not be a basis of masking the field altogether from other browsers, since that loses information. @rfeneon: what do you think?